Submission ran but did not finish its execution due to the errors below. 
This does not mean the program did not execute however, it may or may not have produced some files or none: 

 usage: decrypt_v97159is.py [-h] binary_file output_file
decrypt_v97159is.py: error: argument binary_file: can't open './test_files_encrypt': [Errno 21] Is a directory: './test_files_encrypt'

No files were created within the results folder. (This does not mean files were not created elsewhere)
